A32NX/html_ui/Pages/A32NX_Utils/NXSystemMessages.js

```diff
@@ -1,46 +1,75 @@
+class McduMessage {
+    constructor(_text, _isAmber = false, _isTypeTwo = false, _replace = "") {
+        this.cText = _text;
+        this.isAmber = _isAmber;
+        this.isTypeTwo = _isTypeTwo;
+        this.replace = _replace;
+    }
+
+    /**
+     * `set text(_t) {}` is not allowed to ensure thread safety, editing the original definition should never be allowed
+     * Both NXSystemMessages and NXFictionalMessages messages shall always be readable ONLY
+     */
+
```

Comment:
```suggestion
```

Reply:
I've added this intentional so that everyone knows why we are not having this method allowed.